pandas/tests/test_graphics.py

```diff
@@ -473,8 +473,8 @@ def test_plot(self):
         ax = _check_plot_works(self.ts.plot, style='.', loglog=True)
         self._check_ax_scales(ax, xaxis='log', yaxis='log')
 
-        _check_plot_works(self.ts[:10].plot, kind='bar')
-        _check_plot_works(self.ts.plot, kind='area', stacked=False)
+        _check_plot_works(self.ts[:10].plot.bar)
```

Comment:
maybe have a few duplicates where you use the plot accessor AND the function call (with ``kind``). You may have done this, just checking.